src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java

```diff
@@ -116,4 +116,95 @@ public void testPagination01() throws Exception
       assertEquals(830, total);
    }
 
+   @Test
+   public void testN01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      ObjectNode json = null;
+
+      res = service.get(url("orders?limit=5&n(shipRegion)")).statusOk();
+      json = res.getJson();
+      assertEquals(5, json.find("meta.pageSize"));
+      assertEquals(5, json.getArray("data").length());
```

Comment:
This needs to actually check that the shipregion field is null.

See comments on https://github.com/RocketPartners/rocket-inversion/pull/97 for consistency.